graphql_compiler/compiler/emit_sql.py

```diff
@@ -104,6 +149,61 @@ def traverse(self, vertex_field, optional):
             onclause=(previous_alias.c[edge.from_column] == self._current_alias.c[edge.to_column]),
             isouter=optional)
 
+    def recurse(self, vertex_field, depth):
+        """Execute a Recurse Block."""
+        previous_alias = self._current_alias
+        edge = self._sql_schema_info.join_descriptors[self._current_classname][vertex_field]
+        primary_key = self._sql_schema_info.primary_keys[self._current_classname]
```

Comment:
If we go with the option for supporting (at configuration level only) the composite primary keys situation, I'd just put a `NotImplementedError` for the case where the primary key spans more than one column.